File: sdk/src/main/java/org/p2p/solanaj/rpc/types/BalanceInfo.java
package org.p2p.solanaj.rpc.types;

public class BalanceInfo {
    String depositAddress;
    long amount;
    String mint;
    String owner;
    int decimals;

    @Override
    public String toString() {
        return "BalanceInfo{" +
                "depositAddress='" + depositAddress + '\'' +
                ", amount=" + amount +
                ", mint='" + mint + '\'' +
                ", owner='" + owner + '\'' +
                ", decimals=" + decimals +
                '}';
    }
}

File: app/src/main/java/com/p2p/wowlet/entities/local/BalanceInfo.kt
package com.p2p.wowlet.entities.local

class BalanceInfo(
    var depositAddress: String,
    var amount: Long,
    var mint: String,
    var owner: String,
    var decimals: Int
) {
    override fun toString(): String {
        return "Address: $depositAddress amount: $amount mint: $mint owner: $owner decimals: $decimals"
    }
}